mLRS/Common/hal/rx-hal-R9M-868-f103c8.h

```diff
@@ -276,7 +278,7 @@ const rfpower_t rfpower_list[] = {
     { .dbm = POWER_3_DBM, .mW = 2 },
     { .dbm = POWER_10_DBM, .mW = 10 },
     { .dbm = POWER_20_DBM, .mW = 100 },
-    { .dbm = POWER_23_DBM, .mW = 200 },
+    { .dbm = POWER_24_DBM, .mW = 251 },
```

Comment:
I guess 250 should be accurate enough and potentially less confusing
dito in below occurences

Reply:
Fair enough, will update.